src/components/play/brick/live/LiveStepper.tsx

```diff
@@ -44,6 +44,7 @@ const LiveStepper: React.FC<StepperProps> = ({
     }
     questionIndex++;
     let index = questionIndex;
+    console.log(activeStep, questionIndex);
```

Comment:
@farmanahmed2007 don`t put logs in code

Reply:
I have no reason to put logs I took pull it was there